app/interactors/add_movie_to_round.rb

```diff
@@ -3,26 +3,42 @@
 class AddMovieToRound
   include Hanami::Interactor
 
-  def initialize(import_service: ImportTmdbMovie.new)
-    @import_service = import_service
+  expose :movie, :round
+
+  def initialize(import_tmdb_movie_interactor: ImportTmdbMovie.new)
+    @import_tmdb_movie_interactor = import_tmdb_movie_interactor
   end
 
-  def call(round:, tmdb_id:)
+  def call(round:, tmdb_movie_id:)
```

Comment:
This is minor but I believe it's worth it to care.

One may notice that TMDb stores many entity-sets. For now, we only care about movies and directors, but we store the TMDb ID for both. Of course, if its an attribute of a movie, the "`tmdb_id`" is the ID on that service **for movies**. But outside of that, in a service, `tmdb_id` is not specific enough (we could be mixing here the director ID for whatever reason).